curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c8be904ffff922f3c6f8d0.md

```diff
@@ -19,16 +19,16 @@ Your `getCaloriesFromInputs` function should call your `cleanInputString` functi
 assert.match(getCaloriesFromInputs.toString(), /cleanInputString\(/);
```

Comment:
anything that makes it understandable that the function needs to be invoked within `currVal`, when the user gets it wrong.
```suggestion
You should Invoke the `cleanInputString` function within the `const` named `currVal`.

```js
assert.match(getCaloriesFromInputs.toString(), /cleanInputString\(/);
```
```